psutil/_psutil_windows_uss.c

```diff
@@ -0,0 +1,73 @@
+/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim: set ts=8 sts=2 et sw=2 tw=80: */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "_psutil_windows_uss.h"
+#include <psapi.h>
+
+BOOL
+calc_uss(DWORD target, unsigned long long* aN)
```

Comment:
On a second thought I would prefer you define this in `psutil/_psutil_windows.c` as `proc_memory_uss`.
From the python file (`psutil/_pswindows.py`) after you call `cext.psutil_proc_memory_info` (or `cext.psutil_proc_memory_info_2` you will call `cext.proc_memory_uss` to get USS stats separately.